akka-remote/src/main/mima-filters/2.6.0.backwards.excludes/28300-fix-2.6.0-mima.excludes

```diff
@@ -0,0 +1,5 @@
+# FIXME: Incompatibilities against 2.6.0 / Scala 2.13 is that something we need to look into?
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.EndpointManager#Send.unapply")
```

Comment:
Do we have to look into these? Or can we treat those as internal as well?

Reply:
I think these particular ones are fine: the full incompatibilities are:

Send:
```
2.13.0: (Lakka/remote/EndpointManager$Send;)Lscala/Option<Lscala/Tuple4<Ljava/lang/Object;Lakka/actor/ActorRef;Lakka/remote/RemoteActorRef;Lscala/Option<Lakka/remote/SeqNo;>;>;>;
2.13.1: (Lakka/remote/EndpointManager$Send;)Lscala/Option<Lscala/Tuple4<Ljava/lang/Object;Lakka/util/OptionVal<Lakka/actor/ActorRef;>;Lakka/remote/RemoteActorRef;Lscala/Option<Lakka/remote/SeqNo;>;>;>;
```

OutboundStreamMatValues:
```
2.13.1: (Lakka/remote/artery/Association$OutboundStreamMatValues;)Lscala/Option<Lscala/Tuple3<Lakka/util/OptionVal<Lakka/stream/SharedKillSwitch;>;Lscala/concurrent/Future<Lakka/Done;>;Lakka/util/OptionVal<Lakka/remote/artery/Association$StopSignal;>;>;>;
2.13.0: (Lakka/remote/artery/Association$OutboundStreamMatValues;)Lscala/Option<Lscala/Tuple3<Lakka/stream/SharedKillSwitch;Lscala/concurrent/Future<Lakka/Done;>;Lakka/remote/artery/Association$StopSignal;>;>;
```

AkkaPduCodec#Message:

```
2.13.1: (Lakka/remote/transport/AkkaPduCodec$Message;)Lscala/Option<Lscala/Tuple5<Lakka/actor/InternalActorRef;Lakka/actor/Address;Lakka/remote/WireFormats$SerializedMessage;Lakka/util/OptionVal<Lakka/actor/ActorRef;>;Lscala/Option<Lakka/remote/SeqNo;>;>;>;
2.13.0: (Lakka/remote/transport/AkkaPduCodec$Message;)Lscala/Option<Lscala/Tuple5<Lakka/actor/InternalActorRef;Lakka/actor/Address;Lakka/remote/WireFormats$SerializedMessage;Lakka/actor/ActorRef;Lscala/Option<Lakka/remote/SeqNo;>;>;>;
```

These are the result of the 2.13.0 -> 2.13.1 update, see https://github.com/scala/scala/pull/8127

(now validating this to be extra sure ;) )

Reply:
Send, Association, AkkaPduCodec are all internal

Reply:
Thanks for checking.